Gopkg.toml

```diff
@@ -38,24 +34,34 @@ required = [
 
 [[constraint]]
   name = "github.com/openshift/library-go"
-  revision = "fc4b2d5cbcf0ab855422e340c78fe7a53a38bee3"
+  revision = "9067dee609b10991fa6c26b5d51fe095b2331027"
```

Comment:
Is there a reason for using this rev instead of latest (`dab26bb3a8dc7fccde7227194af755bbff30ce5d`)?

Reply:
Typo on my part, good catch!